zk_toolbox/crates/zk_supervisor/src/commands/test/integration.rs

```diff
@@ -45,7 +45,8 @@ pub async fn run(shell: &Shell, args: IntegrationArgs) -> anyhow::Result<()> {
         "yarn jest --forceExit --testTimeout 120000 -t {test_pattern...}"
     )
     .env("CHAIN_NAME", ecosystem_config.current_chain())
-    .env("MASTER_WALLET_PK", wallets.get_test_pk(&chain_config)?);
+    .env("MASTER_WALLET_PK", wallets.get_test_pk(&chain_config)?)
+    .env("ZKSYNC_DEBUG_LOGS", "true");
```

Comment:
<img width="580" alt="image" src="https://github.com/user-attachments/assets/8b51fd07-1084-45ca-952d-62a30a1b3d4b">
We have it under verbose flag